src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py

```diff
@@ -233,7 +233,7 @@ def get_order_list(self, num_inference_steps: int) -> List[int]:
                     orders = [1, 2, 3] * (steps // 3) + [1, 2]
             elif order == 2:
                 if steps % 2 == 0:
-                    orders = [1, 2] * (steps // 2)
+                    orders = [1, 2] * (steps // 2 - 1) + [1, 1]
```

Comment:
for `order=2` and `lower_order_final=True`, currently it generate orders like `1,2,1,2...,1,2` so final order will be `2`
changing it to have both of the last two final steps to be 1 here

Reply:
Is this only necessary for DPMSingleStepSolver? Not for the other ones?